osmosis-installer.py

```diff
@@ -161,6 +189,142 @@ def cosmovisorInit ():
+    print(bcolors.OKGREEN + "Preparing v7 Upgrade..." + bcolors.ENDC)
+    subprocess.run(["git checkout v7.0.3"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True, env=my_env)
+    subprocess.run(["make build"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True, env=my_env)
+    subprocess.run(["cp build/osmosisd "+osmo_home+"/cosmovisor/upgrades/v7/bin"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True, env=my_env)
+    subprocess.run(["git checkout v3.1.0"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True, env=my_env)
+    subprocess.run(["make install"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True, env=my_env)
+    subprocess.run([". "+HOME+"/.profile"], shell=True, env=my_env)
+    subprocess.run(["cp "+ GOPATH +"/bin/osmosisd "+osmo_home+"/cosmovisor/genesis/bin"], shell=True, env=my_env)
+    print(bcolors.OKGREEN + "Adding Persistent Peers For Replay..." + bcolors.ENDC)
+    peers = "9fd886cd0dd656e01aaedf61db63af1bf79b701e@164.92.138.207:26656"
+    subprocess.run(["sed -i -E 's/persistent_peers = \"\"/persistent_peers = \""+peers+"\"/g' "+osmo_home+"/config/config.toml"], shell=True)
+    subprocess.run(["clear"], shell=True)
+    startReplayNow()
+
+
+def replayFromGenesisRocksDb ():
+    print(bcolors.OKGREEN + "Changing db_backend to rocksdb..." + bcolors.ENDC)
+    subprocess.run(["sed -i -E 's/db_backend = \"goleveldb\"/db_backend = \"rocksdb\"/g' "+osmo_home+"/config/config.toml"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+    print(bcolors.OKGREEN + "Installing rocksdb..." + bcolors.ENDC)
+    print(bcolors.OKGREEN + "This process may take 15 minutes or more" + bcolors.ENDC)
+    os.chdir(os.path.expanduser(HOME))
+    subprocess.run(["sudo apt-get install -y libgflags-dev libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev libzstd-dev"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+    subprocess.run(["git clone https://github.com/facebook/rocksdb.git"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+    os.chdir(os.path.expanduser(HOME+"/rocksdb"))
+    # originally used v6.27.3
+    subprocess.run(["git checkout v6.29.3"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+    subprocess.run(["export CXXFLAGS='-Wno-error=deprecated-copy -Wno-error=pessimizing-move -Wno-error=class-memaccess'"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+    subprocess.run(["sudo make shared_lib"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+    subprocess.run(["sudo make install-shared INSTALL_PATH=/usr"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+    subprocess.run(["sudo echo 'export LD_LIBRARY_PATH=/usr/local/lib' >> $HOME/.bashrc"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+    # subprocess.run(["source $HOME/.bashrc"], shell=True)

```

Comment:
Also, not sure if there is a reason for keeping the dead code?

Reply:
Nope, will remove now!